src/Compilers/Core/Portable/Serialization/ObjectWriter.cs

```diff
@@ -361,10 +361,14 @@ public void Dispose()
             public bool TryGetReferenceId(object value, out int referenceId)
                 => _valueToIdMap.TryGetValue(value, out referenceId);
 
-            public void Add(object value)
+            public void Add(object value, bool isReusable)
             {
                 var id = _nextId++;
```

Comment:
>var id = _nextId++; [](start = 16, length = 19)

Should this code be conditional as well?

Reply:
I did not want to change the logic that dispenses IDs. There is no problem with it and changing would be unnecessary risk at this point.

---
In reply to: [181832602](https://github.com/dotnet/roslyn/pull/26156#discussion_r181832602) [](ancestors = 181832602)